src/components/GrowthChart/GrowthChart.tsx

```diff
@@ -25,38 +27,38 @@ export const GrowthChart = ({
     setDefaultIndicatorError,
 }: GrowthChartProps) => {
     const trackedEntityGender = trackedEntity?.gender;
+    const dateOfBirth = useMemo(() => new Date(trackedEntity.dateOfBirth), [trackedEntity.dateOfBirth]);
```

Comment:
I need to wrap it in a useMemo to avoid this eslint error:
ESLint: The 'dateOfBirth' object construction makes the dependencies of useMemo Hook (at line 31) change on every render. To fix this, wrap the initialization of 'dateOfBirth' in its own useMemo() Hook.(react-hooks/exhaustive-deps)